### PR TITLE
feat(macros): add zh-CN localization in HTTPSidebar.ejs

### PR DESCRIPTION
**Pull request by jasonren0403**
## Summary
Add translations to simplified Chinese in HTTP sidebars.

### Problem
If entered the HTTP topics with zh-CN locale, the sidebars is not localized in pages `https://developer.mozilla.org/zh-CN/docs/Web/HTTP/*`

### Solution
add zh-CN key in sidebar ejs file and fix up translations.

---

### Before

before

### After

after

### CODE REVIEW

**Comment by queengooborg** [comment]
@yin1999 Would you mind reviewing the translations for accuracy?

File: kumascript/macros/HTTPSidebar.ejs [hunk 1]
```diff
@@ -235,6 +235,44 @@ var text = mdn.localStringMap({
...
+  },
+  'zh-CN': {
+    'HTTP': 'HTTP',
+    'HTTPGuide': 'HTTP 指南',
+    'Basics': 'HTTP 基础',
+    'Overview': 'HTTP 概述',
+    'Evolution': 'HTTP 的发展',
+    'ResourcesURI': '资源和 URI',
+    'Identifying': '标识互联网上的内容',
+    'DataURIs': '数据 URI',
```
Comment (yin1999) [inline kumascript/macros/HTTPSidebar.ejs:247]:
```suggestion
    'DataURIs': 'Data URI',
```

File: kumascript/macros/HTTPSidebar.ejs [hunk 2]
```diff
@@ -235,6 +235,44 @@ var text = mdn.localStringMap({
...
+    'MIMETypes': 'MIME 类型介绍',
+    'ListMIMETypes': '常见 MIME 类型列表',
```
Comment (yin1999) [inline kumascript/macros/HTTPSidebar.ejs:249]:
```suggestion
    'ListMIMETypes': '常见的 MIME 类型',
```

File: kumascript/macros/HTTPSidebar.ejs [hunk 3]
```diff
@@ -235,6 +235,44 @@ var text = mdn.localStringMap({
...
+    'Ranges': 'HTTP 请求范围',
+    'Redirects': 'HTTP 重定向',
+    'Conditionals': 'HTTP 条件请求',
+    'ContentNego': 'HTTP 内容协商',
+    'Headers': 'HTTP 头部',
+    'Response_codes': '响应代码',
+    'Compression': 'HTTP 压缩',
+    'Cookies': 'HTTP cookies',
+    'Caching': 'HTTP 缓存',
+    'CORS': 'HTTP 访问控制 (CORS)',
```
Comment (yin1999) [inline kumascript/macros/HTTPSidebar.ejs:263]:
```suggestion
    'CORS': 'HTTP 访问控制（CORS）',
```

File: kumascript/macros/HTTPSidebar.ejs [hunk 4]
```diff
@@ -235,6 +235,44 @@ var text = mdn.localStringMap({
...
+    'ResourcesURI': '资源和 URI',
+    'Identifying': '标识互联网上的内容',
+    'DataURIs': '数据 URI',
+    'MIMETypes': 'MIME 类型介绍',
+    'ListMIMETypes': '常见 MIME 类型列表',
+    'WWWorNotWWW': '选择 www 或非 www 域名？',
+    'Messages': 'HTTP 消息',
+    'Session': '典型的 HTTP 会话',
+    'Connection1x': 'HTTP/1.x 的连接管理',
+    'Ranges': 'HTTP 请求范围',
```
Comment (yin1999) [inline kumascript/macros/HTTPSidebar.ejs:254]: The title of this document is not right:

```suggestion
    'Ranges': 'HTTP 范围请求',
```

**Review by yin1999 — COMMENTED** [review]
Some suggestions of mine

**Comment by yin1999** [comment]
LGTM, it's time to poke @mdn/mdn-community-engagement

**Review by caugner — APPROVED** [review]

**Pull request merged**